elements/pfe-badge/src/pfe-badge.js

```diff
@@ -14,15 +14,48 @@ class PfeBadge extends PFElement {
   }
 
   get schemaUrl() {
-    return "pfe-badge.json";
+    // return "pfe-badge.json";
```

Comment:
Want to just delete the `schemaUrl` entirely?

Reply:
SGTM